hazelcast-client/src/main/java/com/hazelcast/client/cache/impl/AbstractClientCacheProxy.java

```diff
@@ -429,11 +430,11 @@ private void waitResponseFromAllPartitionsForPutAll(List<FutureEntriesTuple> fut
              *        In this test exception is thrown at `CacheWriter` and caller side expects this exception.
              * So as a result, we only throw the first exception and others are suppressed by only logging.
              */
-            ExceptionUtil.rethrow(error);
+            throw rethrow(error);
         }
     }
 
-    private void handleNearCacheOnPutAll(List<Map.Entry<Data, Data>> entries, boolean invalidate) {
```

Comment:
It was an unused parameter on a private method. The whole method was empty until November 2015, but it looks like the parameter was never used.